2227/2/lib/rapper_analizator.rb

```diff
@@ -0,0 +1,52 @@
+# Analysis of rappers statistic
+class RapperAnalyzer
+  DEFAULT_ROUNDS_COUNT = 3
+
+  def self.calculate_statistic(name)
+    new(name).fetch_statistic
+  end
+
+  def initialize(name)
+    all_battles = DataStorage.find_all_battles(name)
```

Comment:
so you go through the list of files n + n * m times (m is number of rapers and n stands for files)
It's possible to do this in one go.